File: elemental/src/dto/fotodto.ts
import {UserDTO} from "./userdto";

export class FotoDTO {
  idFoto: number

  url: string

  user:UserDTO

  constructor(idFoto: number,

  url: string,

  user:UserDTO) {
    this.idFoto = idFoto;
    this.url = url;
    this.user = user;
  }
}

File: elemental-service/src/main/java/com/simonecampis/Elemental/dto/FotoDTO.java
package com.simonecampis.Elemental.dto;

import com.simonecampis.Elemental.model.User;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FotoDTO {

    private Long idFoto;

    private String url;

    private User user;
}
